src/kOS/Binding/FlightControlManager.cs

```diff
@@ -201,6 +201,8 @@ public void SelectAutopilotMode(object autopilotMode)
             if (autopilotMode is Direction)
             {
                 //TODO: implment use of direction subclasses.
+                throw new Safe.Exceptions.KOSException(
```

Comment:
Yeah, that was the goal a long time ago.  The problem is that the actual "direction" of prograde changes, even though the stored variable stays the same (sasmode uses `set` not `lock`).  Then we wanted to have a special way to identify these magic directions, so the bound variable `prograde` would include a flag that said "I was made to be a magic updating variable".  But that got complicated, and there isn't a good way to do "stability" the same way, nor anti target.  So we kind of gave up.